test/auto_ownership/configs/auto_ownership.csv

```diff
@@ -0,0 +1,64 @@
+#,student mandatory tour rail mode portion,@((setting('use_accessibility') == 1) & (##studentsRailProportion##)),0.26285636,0.26285636,0.26285636,,,,0,0,0,0,0
+,ASC adjustment for non-GQ PRE-AO,@df.hh_unit_type==1,,0.04582774,0.04582774,-1.602468896,-1.602468896,-1.602468896,-2.169791463,-2.169791463,-2.169791463,-2.169791463,-2.641133846
+,ASC adjustment for non-GQ POST-AO,@((df.hh_unit_type==1) & (setting('use_accessibility') == 1)),,-0.013942306,-0.013942306,-0.132328921,-0.132328921,-0.132328921,-0.180285245,-0.180285245,-0.180285245,-0.180285245,-0.186095557
+,adjustment for 2 workers in non-family 0-car household,@((df.hh_type>3) & (df.num_workers==2)),0.725034403,,,,,,,,,,
+,adjustment for 3 workers in non-family 0-car household,@((df.hh_type>3) & (df.num_workers==3)),1.194820674,,,,,,,,,,
+,adjustment for 4+ workers in non-family 0-car household,@((df.hh_type>3) & (df.num_workers>3)),2.844226409,,,,,,,,,,
+,adjustment for 4+ workers in family 0-car household,@((df.hh_type<=3) & (df.num_workers>3)),2.116557339,,,,,,,,,,
+,TM2 Round 2 ASC adjustment for 1 workers in 0-car household,@df.num_workers==1,1.097234124,,,-0.151698573,-0.151698573,-0.151698573,-0.209786195,-0.209786195,-0.209786195,-0.209786195,-0.308288035
+,TM2 Round 2 ASC adjustment for 2 workers in 0-car household,@df.num_workers==2,0.893156568,,,-0.17099367,-0.17099367,-0.17099367,-0.382492356,-0.382492356,-0.382492356,-0.382492356,-0.551034746
+,TM2 Round 2 ASC adjustment for 3+ workers in 0-car household,@df.num_workers>=3,0.937133013,,,-0.553211417,-0.553211417,-0.553211417,0.065128979,0.065128979,0.065128979,0.065128979,-0.048670057
+,TM2 Round 2 ASC adjustment for non-GQ PRE-AO,@df.hh_unit_type==1,-1.511025686,0,0,0.723793144,0.723793144,0.723793144,0.617090781,0.617090781,0.617090781,0.617090781,0.69421287
+,TM2 Round 2 ASC adjustment for non-GQ POST-AO,@((df.hh_unit_type==1) & (setting('use_accessibility') == 1)),-0.115788174,0,0,0.16375467,0.16375467,0.16375467,0.222715484,0.222715484,0.222715484,0.222715484,0.2639107
+,AV ASC - Household Income under $50k,@df.income<50000,,,-1,,-1,-1,,-1,-1,-1,
+,AV ASC - Household Income 100k+,@df.income>=100000,,,1,,1,1,,1,1,1,
+,AV ASC - Young household,@(df.num_persons_18_to_35>=df.num_persons_65_plus),,,0.5,,0.5,0.5,,0.5,0.5,0.5,
+,AV ASC - Old household,@(df.num_persons_18_to_35 < df.num_persons_65_plus),,,-1,,-1,-1,,-1,-1,-1,
+#,"AV  Coeff Per Hour of travel by auto for work, summed across all workers in household",@((setting('use_accessibility') == 1) & (##@workAutoTime##)),,,0.25,,0.25,0.25,,0.25,0.25,0.25,
+,AV ASC - 100% AV Share,@setting('mobility_av_share')==1.0,,-999,,-999,,-999,-999,,-999,-999,-999
+,AV ASC - 90% AV Share,@setting('mobility_av_share')==0.9,,,,,,,,,,,
+,AV ASC - 50% AV Share,@setting('mobility_av_share')==0.5,,,,,,,,,,,
+,AV ASC - 10% AV Share,@setting('mobility_av_share')==0.1,,,,,,,,,,,
```

Comment:
Can we simply these by just adding one line that checks if av share is greater than 0 or less than 1, then  do not turn off any alternatives?